datafusion-cli/src/command.rs

```diff
@@ -34,6 +35,8 @@ pub enum Command {
     Help,
     ListTables,
     DescribeTable(String),
+    FunctionList,
```

Comment:
how about using `ListFunctions` to keep it consistent?

Reply:
yup, was going to update that